frontend/renderer/app.ts

```diff
@@ -27,7 +27,17 @@ function createApp(pageContext: PageContext, isClient = true) {
         DefaultApolloClient,
         createApolloClient(pageContext.publicEnv.ENDPOINTS)(getToken, isClient),
       )
-      provide('authService', new AuthService(pageContext.publicEnv.AUTH))
+      try {
+        provide('authService', new AuthService(pageContext.publicEnv.AUTH))
+      } catch (error) {
+        if (String(error).includes('SecurityError: Failed to read')) {
+          toast.error(i18n.global.t('error.enablecookie'), {
+            autoClose: false,
+            closeButton: false,
+            closeOnClick: false,
+          })
+        }
+      }
```

Comment:
```suggestion
      try {
        provide('authService', new AuthService(pageContext.publicEnv.AUTH))
      } catch (error) {
        if (!error instancof WellKnownErrorClass) {
          throw error
        }
        toast.error(i18n.global.t('error.enablecookie'), {
          autoClose: false,
          closeButton: false,
          closeOnClick: false,
        })
      }
```

You also need to re-throw the error, if it's not the expected error!

Reply:
@roschaefer I think here is the problem we cannot find the class of `Security Error` - thats some browser related stuff

Reply:
It's OK! Then use `String(error).includes('SecurityError: Failed to read')` I just guess it's different for different browsers.

Important is re-throwing the error if the condition doesn't match. It's a shortcoming of JavaScript that `catch(err)` does not filter for error classes and this is an eternal source of bugs.

Reply:
@ulfgebhardt The error is documented here: https://developer.mozilla.org/en-US/docs/Web/API/DOMException#securityerror